### ISSUE TITLE
[Bug]: Error on form input not announced to screen readers

### ISSUE THREAD

**Issue by cjrace** [dfe-analytical-services/shinyGovstyle#192]
### Overview

<img width="478" height="310" alt="Image" src="https://github.com/user-attachments/assets/58b7784d-f283-46d5-94cb-e2c25cfe7e0f" />

The service page has two textarea components. The second instance of a textarea in the section is used to test error handling. Activating the button ‘Check for errors’ causes an error message to appear. The error message reads ‘Please do not leave blank’. Those that use screen-reading software will not hear the software announce the error, and this is because the error message is not added to an element that uses the role=alert attribute.

### Steps to Reproduce

Trigger the error on the showcase app, inspect the HTML.

### Expected Behavior

Screen reader comments:

> “Once submitting the form with errors using JAWS, I became confused as no indication of any error handling was identified once the page had loaded. I found by using various methods of navigation, that errors were identified, however they do not follow the conventions of the Gov UK Design System. Including a clear error handling process will resolve this issue, causing less confusion for users of screen reading software."

We need to add

### Additional Information

Can review against the GOV.UK error message component - https://design-system.service.gov.uk/components/error-message/.

Page 38 of the September 2025 accessibility report.
